internal/core/application/account_service.go

```diff
@@ -70,90 +74,90 @@ func (as *AccountService) CreateAccountBIP44(
 }
 
 func (as *AccountService) DeriveAddressesForAccount(
-	ctx context.Context, accountName string, numOfAddresses uint64,
+	ctx context.Context, namespace string, numOfAddresses uint64,
 ) (AddressesInfo, error) {
 	if numOfAddresses == 0 {
 		numOfAddresses = 1
 	}
 
 	addressesInfo, err := as.repoManager.WalletRepository().
-		DeriveNextExternalAddressesForAccount(ctx, accountName, numOfAddresses)
+		DeriveNextExternalAddressesForAccount(ctx, namespace, numOfAddresses)
```

Comment:
Finally, also the repo should allow referring to an account via label or namespace and this should be completely abstracted for the app layer so that we just need to change a variable's name:
```suggestion
		DeriveNextExternalAddressesForAccount(ctx, name, numOfAddresses)
```